src/ToolingSharedLibrary/Includes/CodeGeneration/Contants.h

```diff
@@ -106,7 +106,10 @@ R"(HRESULT hr = VbsEnclaveABI::Enclave::CallVtl1ExportFromVtl1<ParamsT, decltype
 R"(HRESULT hr = VbsEnclaveABI::HostApp::CallVtl0CallbackImplFromVtl0<ParamsT, ReturnParamsT, decltype({}_Abi_Impl)>(function_context, {}_Abi_Impl);)";
 
     static inline constexpr std::string_view c_vtl1_call_to_vtl0_callback =
-R"(THROW_IF_FAILED((VbsEnclaveABI::Enclave::CallVtl0CallbackFromVtl1<ParamsT, ReturnParamsT>({}, flatbuffer_builder, function_result)));)";
+"\n            VbsEnclaveABI::Enclave::CallVtl0CallbackFromVtl1<void>(in_flatbufferT, {});";
+
+    static inline constexpr std::string_view c_vtl1_call_to_vtl0_callback_with_return =
+"\n            auto return_params = VbsEnclaveABI::Enclave::CallVtl0CallbackFromVtl1<{}>(in_flatbufferT, {});";
```

Comment:
><{}>(in_flatbufferT, {});"; [](http://example.com/codeflow?start=84&length=27)

nit: this doesn't need to be parameterized if you do this

    using X = FuncWithAllArgs_1_args;
    auto constexpr Y = "CodeGenTest::AbiDefinitions::FuncWithAllArgs_1_Generated_Stub";

    // ... DT convs

    auto return_params = VbsEnclaveABI::Enclave::CallVtl0CallbackFromVtl1<X>(in_flatbufferT, Y);";